### ISSUE TITLE
VoiceOver on macOS doesn't read all audio input channels

### ISSUE THREAD

**Issue by michaeldessen** [sonosaurus/sonobus#136]
I was remotely assisting a blind person using Sonobus and we discovered that when he used the Voiceover feature to read the menu items, etc., under "Active input channels" it only read the first few of his MOTU interface input channels then stopped, and didn't read the full list, meaning he had no way to activate the SPDIFs that are later down the list. This was on a 2018 Mac mini on 10.15.7. I don't have an interface with enough channels to see if I can reproduce the problem but wanted to mention it in case anyone can help with this. Thanks!
